single-audit/resolve_findings/templates/resolve_findings/base.html

```diff
@@ -0,0 +1,242 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Finding resolution page</title>
+  {% load static %}
+  <link rel="stylesheet" href="{% static 'uswds.min.css' %}">
+  <link href="https://fonts.googleapis.com/css?family=Merriweather:400,900i&display=swap" rel="stylesheet">
+  <title></title>
+</head>
+<body>
+  <a class="usa-skipnav" href="#finding_summary_header">Skip to main content</a>
+  <div class="usa-banner">
+    <div class="usa-accordion">
+      <header class="usa-banner__header">
+        <div class="usa-banner__inner">
+          <div class="grid-col-auto">
+            <img class="usa-banner__header-flag" src="assets/uswds-2.0.2/img/us_flag_small.png" alt="U.S. flag">
```

Comment:
@alexsoble Consider omitting the `uswds-2.0.2` element of this directory structure to help these references stay relatively evergreen.

Reply:
Changed to `/uswds`, good feedback. 😄